### ISSUE TITLE
CI not running on FreeBSD

### ISSUE THREAD

**Issue by asomers** [bytecodealliance/cap-std#340]
PR #74 added a CI configuration for FreeBSD.  However, it isn't running anymore.  Is the Cirrus CI app enabled for this repository in Github Marketplace?
https://cirrus-ci.org/guide/quick-start/

**Comment by sunfishcode** [comment]
I believe Cirrus CI is not enabled for this repository. I'll look into seeing if we can enable it.

**Comment by asomers** [comment]
@sunfishcode it's not working again.  Did you check that Cirrus is enabled?

**Comment by sunfishcode** [comment]
No, Cirrus is not yet enabled. Unfortunately, it has turned out to be not straightforward to enable in this organization.

I am able to run the CI script in a fork of this repo. Doing this just now turned up some issues with the CI script being out of date. I've now fixed those issues in https://github.com/bytecodealliance/cap-std/pull/355 and can confirm that the CI tests all pass again.
